backend/.dockerignore

```diff
@@ -1,23 +1,23 @@
-**/.classpath

```

Comment:
Is this change doing anything? if not, it is a good practice to roll back this change and not push it in the branch. Try to reduce the unnecessary changes next time! This time, I think it is enough and I'll approve it.